docs/llm-hosting/bench/concsweep_real.py

```
@@ -0,0 +1,57 @@
+    }).encode()
+    t0 = time.perf_counter()
+    try:
+        rq = urllib.request.Request(URL, data=body,
+                                    headers={"Content-Type": "application/json"})
+        with urllib.request.urlopen(rq, timeout=1200) as r:
+            d = json.load(r)
+        el = time.perf_counter() - t0
+        with lk:
+            out.append((d["usage"]["completion_tokens"], el))
+    except Exception:
+        pass
```

Ruff (0.16.3)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)